packages/devextreme/js/ui/form/ui.form.js

```diff
@@ -1182,10 +1183,13 @@ const Form = Widget.inherit({
     },
 
     _refresh: function() {
-        const editorSelector = `.${FOCUSED_STATE_CLASS} > :not(.dx-dropdowneditor-input-wrapper) input,`
-            + ` .${FOCUSED_STATE_CLASS} textarea`;
+        const editorSelector = `.${FOCUSED_STATE_CLASS} input, .${FOCUSED_STATE_CLASS} textarea`;
 
-        eventsEngine.trigger(this.$element().find(editorSelector), 'change');
+        this.$element().find(editorSelector).each((idx, el) => {
+            if(!$(el).parents(UNREFRESHABLE_COMPONENT_CLASS).length) {
+                eventsEngine.trigger(el, 'change');
+            }
```

Comment:
Unfortunately it seems to me that this solution is not applicable because it will prevent triggering "change" event for cases when direct dropDownEditor is focused.
Scenario: 
- add a SelectBox with `acceptCustomValue=true` to form
- start typing
- minimize window
- value (as far as i understand is lost)

Please, consider other ddEditors and `acceptCustomValue=true/false` and add tests for these cases